src/docs/arch-glossary.dox

```diff
@@ -22,13 +22,11 @@ they are used in other settings.
 <tr><td>block<td>block manager<td>the smallest granularity of data that is read from or written to a Btree file.  A btree page, when it appears on disk, may consist of one or more blocks.
 <tr><td>block compression<td>btree<td>after pages of btrees are encoded to be written, they may be compressed according to a configured algorithm.  Compressors may be added to WiredTiger via the extension mechanism.
 <tr><td>block manager<td>block manager<td>an internal API that abstracts writing, reading and allocating blocks to and from files.  A block manager is associated with a Btree.
-<tr><td>Bloom filter<td>lsm<td>a data structure that identifies that a key cannot be present.  Used by LSM.
```

Comment:
We still support bloom filters so I think we only need to drop "Used by LSM." here.

Reply:
Okay sure. Added it back in [03bf29c](https://github.com/wiredtiger/wiredtiger/pull/11299/commits/03bf29c20b2d416b3e304477b5cecc0c684ebe9f)